pyins/tests/test_transform.py

```diff
@@ -16,12 +20,93 @@ def test_lla_to_ecef():
     assert_allclose(r_e, [[earth.A + 10, 0, 0], [0, 0, -b + 10]], atol=1e-9)
 
 
+def test_lla_to_ned():
+    lla = [[0, 0, 1000], [90, 90, 0], [0, -90, -1000]]
+    ned = transform.lla_to_ned(lla)
+    a = earth.A + 1000
+    b = (1 - earth.E2) ** 0.5 * earth.A
+    test = [[0, 0, 0], [b, 0, a], [0, -earth.A + 1000, a]]
+    assert_allclose(ned, test, atol=1e-8)
+    assert(isinstance(ned, np.ndarray))
+
+    lla = pd.DataFrame(data=lla, columns=LLA_COLS)
+    ned = transform.lla_to_ned(lla)
+    test = pd.DataFrame(data=test, columns=NED_COLS)
+    assert_allclose(ned, test, atol=1e-8)
+    assert(isinstance(ned, pd.DataFrame))
+
+    lla = [[0, 0, 1000], [90, 90, 0], [0, -90, -1000]]
+    ned = transform.lla_to_ned(lla, [0, -90, -1000])
+    a = earth.A - 1000
+    test = [[0, earth.A + 1000, a], [b, 0, a], [0, 0, 0]]
+    assert_allclose(ned, test, atol=1e-8)
+    assert(isinstance(ned, np.ndarray))
+
+    lla = pd.DataFrame(data=lla, columns=LLA_COLS)
+    ned = transform.lla_to_ned(lla, [0, -90, -1000])
+    test = pd.DataFrame(data=test, columns=NED_COLS)
+    assert_allclose(ned, test, atol=1e-8)
+    assert(isinstance(ned, pd.DataFrame))
+
+
 def test_perturb_ll():
     lla = [40, 50, 0]
     lla_new = transform.perturb_lla(lla, [10, -20, 5])
     lla_back = transform.perturb_lla(lla_new, [-10, 20, -5])
     assert_allclose(lla_back, lla, rtol=1e-11)
 
+    lla = [[40, 50, 0], [-40, 50, 10]]
+    lla_new = transform.perturb_lla(lla, [10, -20, 5])
+    lla_back = transform.perturb_lla(lla_new, [-10, 20, -5])
+    assert_allclose(lla_back, lla, rtol=1e-11)
+
+
+def test_translate_trajectory():
+    traj, imu = sinusoid_velocity_motion(0.1, 60, [10, 20, -3], [5, 7, 3])
```

Comment:
Please rebase  on the latest 1.0 branch, I've adjusted things (including this function name).